### ISSUE TITLE
Examine re-enabling reposts on odysee homepage

### ISSUE THREAD

**Issue by kauffj** [lbryio/lbry-desktop#5894]
- Must not show duplicates
- Put test version up on dev server for evaluation by growth team

**Comment by kauffj** [comment]
this probably requires SDK changes to do

**Comment by kauffj** [comment]
@shyba says that if `limit_claims_per_channel` is passed that this already happens

**Comment by infinite-persistence** [comment]
- Seems like already working:
    - ![image](https://user-images.githubusercontent.com/64950861/116547620-b75ad700-a925-11eb-8067-61650e557b9e.png)
- Also double-checked the code.  We are setting `limit_claims_per_channel` for the homepages.

**Comment by kauffj** [comment]
it has to be done in a way that the same claim doesn't show up twice, even if it is reposted

if it's not feasible to do cleanly with existing APIs then it should be pushed to SDK team to modify APIs to support this

**Comment by kauffj** [comment]
also, I believe all homepage search queries for odysee.com need to be modified to allow reposts as a type, but this should only be done if repetition problem is addressed

**Comment by kauffj** [comment]
@shyba please comment when this is workable (i.e. live on wallet servers)

**Comment by kauffj** [comment]
On hold pending #6180

**Comment by tzarebczan** [comment]
This can be worked on currently, not blocked by 6180. The separate changes were reverted on SDK for now. 

The SDKs / hubs were updated last night and now have this param:

```           
 --remove_duplicates             : (bool) removes duplicated content from search by picking either the
                                                     original claim or the oldest matching repost
```

We probably want this enabled by default on all claim searches.

**Comment by kauffj** [comment]
- Restore reposts as being allowed on homepage claim_searches
- Add `remove_duplicates` flag
